flint.ui/src/components/GCBM/Configurations/Pools.vue

```diff
@@ -1,27 +1,39 @@
 <template>
   <div class="py-4 px-8 text-earth">
-    <a-typography-title><span class="font-normal text-earth"> Configure Pools </span></a-typography-title>
+    <a-typography-title
+      ><span class="font-normal text-earth">
+        Configure Pools
+        <a-input-search
+          v-model:value="searchValue"
+          placeholder="Search for a pool"
+          style="width: 250px; margin-left: 28rem; margin-top: 2px"
+          class="mb-6"
+        /> </span
+    ></a-typography-title>
     <a-typography-text>
-      <span class="text-earth mb-2 block"> Configure the starting points for each pool. </span>
+      <span class="text-earth mb-2 block" style="margin-bottom: 20px">
+        Configure the starting points for each pool.
+      </span>
     </a-typography-text>
-    <a-input-search v-model:value="searchValue" placeholder="Search for a pool" style="width: 250px" class="mb-6" />
     <a-row :gutter="[16, 16]">
       <a-col v-for="(pool, poolKey) in filteredPools" :key="poolKey" :span="16" :lg="{ span: 12 }" :xl="{ span: 8 }">
-        <div class="flex items-center">
+        <div class="span items-center">
           <a-typography-title :level="5" style="margin: 0">
-            <span class="font-normal mr-4"> {{ poolKey }}: </span>
+            <span class="font-normal mr-4"><b>{{ poolKey }}:</b></span>
```

Comment:
for making this bold you can just remove the `font-normal` class and remove the `b` tags, as `a-typography-title` is bold by default

Reply:
ok